tools/change_case/.shed.yml

```diff
@@ -6,5 +6,6 @@ long_description: |
   or lowercase.
 name: change_case
 owner: devteam
+homepage_url: ''
```

Comment:
If there's no relevant homepage, it's fine to leave this out, `planemo shed_lint` gives just a warning.

```suggestion
```